src/components/Kite/Kite.tsx

```diff
@@ -0,0 +1,131 @@
+import React, { useEffect, CSSProperties } from 'react';
+import classNames from 'classnames';
+import { ClassValue } from 'classnames/types';
+
+import ReactDOM from 'react-dom';
+import { CSSTransition } from 'react-transition-group';
+import styles from './Kite.module.css';
+import Button from '../Button';
+import Icon from '../Icon';
+
+interface KiteProps {
+  children?: React.ReactNode;
+  className?: ClassValue;
```

Comment:
Imo, we shouldn't use `ClassValue`.
I'm removing it in an upcoming PR, hopefully.

Our `className` type should just be an optional string.

(Using `ClassValue` would allow users to pass anything that the `classnames` package can handle, e.g. arrays, objects with optional extra class names. That's fine, and might be nice to have, but 1) we don't document it, 2) it causes some TS headaches, and 3) I think consumers can just use the `classnames` package on their own and send us full composed strings.